test/jdk/java/util/concurrent/Executors/AutoShutdown.java

```diff
@@ -110,6 +123,21 @@ void testActiveWorker(Supplier<ExecutorService> supplier,int queuedTaskCount) th
         assertEquals(ntasks, completedTaskCount.get());
     }
 
+    @ParameterizedTest
+    @MethodSource("shutdownMethods")
+    void testShutdownUnlinksCleaner(Consumer<ExecutorService> shutdown) throws Exception {
+        ClassLoader classLoader = Utils.getTestClassPathURLClassLoader(ClassLoader.getPlatformClassLoader());
+
+        ReferenceQueue<?> queue = new ReferenceQueue<>();
+        Reference<?> reference = new PhantomReference(classLoader, queue);
+
+        classLoader.loadClass("AutoShutdown$IsolatedClass").getDeclaredMethod("shutdown", Consumer.class).invoke(null, shutdown);
```

Comment:
Something like this:
```suggestion
        classLoader.loadClass("AutoShutdown$IsolatedClass")
                            .getDeclaredMethod("shutdown", Consumer.class)
                            .invoke(null, shutdown);
```